3711/1/parser/repo.rb

```diff
@@ -0,0 +1,82 @@
+require_relative '../ruby_gem'
+require 'json'
+require 'net/http'
+require 'nokogiri'
+require 'open-uri'
+
+module Parser
+  class Repo
+    CLIENT_ID = '26b9a6fc285e836c9f7a'.freeze
+    CLIENT_SECRET = '7ca0b9c0e86047afbac0951d3d505bf7801a5603'.freeze
```

Comment:
I would get rid of this configs here.
It's possible to generate one token.
Consider to put token in some config file